nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw soundcloud podcast display function 117

# Description
#117 Added function that displays podcast player on the modal only if the users.pod property has a SoundCloud link
#137 Learn More button changes URL to #{anchortag}

## Type of change


# Testing Instructions

- Pull down the branch and serve web site in a browser
- On a Developer Card, click **Learn More** button. The URL should not change, and the modal should open.
- In the student Modal, no SoundCloud player should be visible because no C51 students have a podcast episode added to the users.pod property in the database. 


ADDITION:
Added a "More Details" button on the Event Card.
Added styling to fall in-line with existing buttons.

# Checklist:

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] My changes generate no new warnings or errors
- [] I have added test instructions that prove my fix is effective or that my feature works
